payments-core/src/test/java/com/stripe/android/view/CardFormViewTest.kt

```diff
@@ -358,6 +376,18 @@ internal class CardFormViewTest {
         }
     }
 
+    @Test
+    fun `Returns the correct create params when user selects preferred network`() {
```

Comment:
do we have any tests for when a user selects a network which isn't the merchant's preferred network?

Reply:
I added tests for brand scenarios to [CardBrandViewTest](https://github.com/stripe/stripe-android/blob/8dea23c73cc09c3c395948f3dcf9b24c70b7483e/payments-core/src/test/java/com/stripe/android/view/CardBrandViewTest.kt) as it's responsible for creating the network params

Reply:
have you pushed those changes?

Reply:
Yh, it's a new file

Reply:
I don't see a test for the scenario I meant. Here's what I'm thinking of:
- merchant has preferred network of cartes bancaires
- user uses the card brand choice drop down to select mastercard as their card brand
- the card should be created with mastercard

Can you add a test for this? My understanding is that the user's choice should override the merchant's preferred network, is that correct?